src/core/mac/data_poll_handler.cpp

```diff
@@ -272,12 +272,17 @@ void DataPollHandler::HandleSentFrame(const Mac::TxFrame &aFrame, Error aError,
             {
                 uint32_t frameCounter;
                 uint8_t  keyId;
+                Error   error;
```

Comment:
To fix pretty check.
```suggestion
                Error    error;
```